packages/powersync_sqlcipher/pubspec.yaml

```diff
@@ -0,0 +1,31 @@
+name: powersync_sqlcipher
+version: 0.0.1
```

Comment:
Can we start at `0.1.0`? I generally view 0.0.x versions as "very experimental".

Reply:
Will do with melos version once the issues are resolved.